src/dmd/parse.d

```diff
@@ -5528,6 +5528,90 @@ final class Parser(AST) : Lexer
 
     }
 
+    /***
+     * Parse an assignment condition for if or while statements.
+     *
+     * Params:
```

Comment:
Leftover from a previous version. Need to update to `Returns`, but waiting to see what the testsuite thinks.